mediasms.go

```diff
@@ -0,0 +1,114 @@
+package mediasms
+
+import (
+	"bytes"
+	"encoding/json"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"strings"
+
+	"github.com/comvex-jp/mediasms-go/errors"
+	"github.com/comvex-jp/mediasms-go/models"
+	"github.com/comvex-jp/mediasms-go/translations"
+)
+
+// Client struct
+type Client struct {
+	Username string
+	Password string
+	Prefix   string
+}
+
+// resultCode from media4u
+type resultCode struct {
+	Result string `json:"result"`
+}
+
+func createSMSID(prefix, messageID string) string {
+	return prefix + messageID
+}
+
+// SMSURL constant for sms-console
+const SMSURL = "https://www.sms-console.jp/"
+
+// Send request to media4u
+func (c Client) Send(messageID string, val models.SendRequest) (models.APIResponse, error) {
+	smsID := createSMSID(c.Prefix, messageID)
+
+	val.SMSID = smsID
+
+	jsonValue, _ := json.Marshal(val)
+	client := &http.Client{}
```

Comment:
`client` and `c` are very similar on this context. Can you be more specific on the variables name? `httpClient` and `client` i think it will be easier to understand.